static/js/containers/SettingsPage.js

```diff
@@ -23,7 +23,7 @@ class SettingsPage extends ProfileFormContainer {
 
   render() {
     const { profiles } = this.props;
-    let props = Object.assign({}, this.profileProps(profiles[SETTINGS.user.username]));
+    let props = this.profileProps(profiles[SETTINGS.user.username]);
```

Comment:
Yes, `props` is only used as `{...props}` so there's already a shallow copy going on